app/javascript/utilities/document_ready.js

```diff
@@ -0,0 +1,7 @@
+Document.prototype.ready = new Promise((resolve) => {
```

Comment:
nice! Smart clean way to wait for the right moment to convert ids to names without bringing all that '$' into it :D